src/main/kotlin/app/revanced/patches/music/ad/video/HideVideoAds.kt

```diff
@@ -25,23 +26,23 @@ import app.revanced.util.exception
     ],
 )
 @Suppress("unused")
-object HideMusicVideoAds : BytecodePatch(
-    setOf(ShowMusicVideoAdsParentFingerprint),
+object HideVideoAds : BytecodePatch(
+    setOf(ShowVideoAdsParentFingerprint),
 ) {
     override fun execute(context: BytecodeContext) {
-        ShowMusicVideoAdsParentFingerprint.result?.let {
-            val showMusicVideoAdsMethod = context
+        ShowVideoAdsParentFingerprint.result?.let {
+            val showVideoAdsMethod = context
                 .toMethodWalker(it.mutableMethod)
                 .nextMethod(it.scanResult.patternScanResult!!.startIndex + 1, true).getMethod() as MutableMethod
 
-            showMusicVideoAdsMethod.addInstruction(0, "const/4 p1, 0x0")
-        } ?: throw ShowMusicVideoAdsParentFingerprint.exception
+            showVideoAdsMethod.addInstruction(0, "const/4 p1, 0x0")
+        } ?: throw ShowVideoAdsParentFingerprint.exception
     }
 }
 
-@Deprecated("This patch class has been renamed to HideMusicVideoAds.")
-object MusicVideoAdsPatch : BytecodePatch(
-    dependencies = setOf(HideMusicVideoAds::class),
+@Deprecated("This patch class has been renamed to HideVideoAds.")
+object VideoAdsPatch : BytecodePatch(
```

Comment:
Do I simply have to rename here?
```suggestion
object MusicVideoAdsPatch : BytecodePatch(
```